src/Bicep.Core/Emit/ExpressionConverter.cs

```diff
@@ -78,6 +78,11 @@ public LanguageExpression ConvertExpression(SyntaxBase expression)
                         function.Name.IdentifierName,
                         function.Arguments.Select(a => ConvertExpression(a.Expression)).ToArray());
 
+                case InstanceFunctionCallSyntax instanceFunctionCall:
+                    return ConvertFunction(
+                        instanceFunctionCall.Name.IdentifierName,
+                        instanceFunctionCall.Arguments.Select(a => ConvertExpression(a.Expression)).ToArray());
```

Comment:
Could you either assert here that the base type is a namespace, or add a comment to explain this?

Reply:
Makes sense, added an assert here using SemanticModel.GetSymbolInfo to get its symbol information, if is not a NamespaceSymbol, the code throws an argumentException